InterGalacticCurrencyConvertor/src/main/java/query/processor/InterGalacticToDecimalQueryProcessor.java

```diff
@@ -0,0 +1,72 @@
+package main.java.query.processor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import main.java.numerals.InterGalacticNumerals;
+import main.java.numerals.RomanNumerals;
+import main.java.query.processor.QueryProcessor;
+import main.java.DataProcessResult;
+import main.java.ProcessorData;
+
+/**
+ * Processes query of the form "how much is x ?" where "x" denotes a number
+ * of InterGalactic numerals separated by white-spaces.
+ */
+public class InterGalacticToDecimalQueryProcessor implements QueryProcessor {
+  public boolean canProcess(ProcessorData pData) {
+    Objects.requireNonNull(pData, "Invalid processor data");
+    String regex = "^\\s*how\\s+much\\s+is\\s+\\b.+\\b\\s+\\?\\s*$";
```

Comment:
Store all the regexes as constants in this class.

Reply:
done